### ISSUE TITLE
experimental branch - ofxSingleSoundPlayer - load via getBuffer()

### ISSUE THREAD

**Issue by stephanschulz** [roymacdonald/ofxSoundObjects#39]
I am trying to load each player by using a file buffer `file_0.getBuffer()` instead of `load(dir.getPath(i))`, but app crashes right away.

```

 ofxSoundFile file_0;
 file_0.load(dir.getPath(0));
        
 if(players[startIndex + i]->load( file_0.getBuffer(), ofToString(i))){

}
```
<img width="1447" alt="Screen Shot 2021-05-07 at 10 17 07 AM" src="https://user-images.githubusercontent.com/1054816/117463211-6cd8ec00-af1d-11eb-99b5-7f7429259f65.png">

**Comment by roymacdonald** [comment]
Yes. I think that this generates a data race which makes it crash.

**Comment by roymacdonald** [comment]
sure. I will double check what is going on.

**Comment by roymacdonald** [comment]
@stephanschulz I fixed it. please test it and let me know. 
I was not able to download the project you linked. Seems like the file got corrupt or something
